data/en/getlocaleinfo.json

```diff
@@ -4,29 +4,34 @@
 	"syntax":"getLocaleInfo()",
 	"returns":"struct",
 	"related":["getLocaleDisplayName","getLocale","getLocaleCountry","getLocaleLanguage"],
-	"description":"A short description that describes what the tag or function does.",
-	"discouraged":"Only add this key if this tag/function is discouraged by the community.",
+	"description":"Returns a structure containing info to a specific locale.\r\n\r\nThis function combines the locale functions getLocaleCountry, getLocaleDisplayName, and getLocaleLanguage to a single function.",
 	"params": [
-		{"name":"locale","description":"Geographic/language locale value","required":true,"default":"getLocale()","type":"string","values":["en","de_DE","de_ch","..."]},
-		{"name":"displayLocale","description":"Locale's display name/Output language","required":false,"default":"getLocaleDisplayName()","type":"string","values":["English","German","French","..."]}
+		{"name":"locale", "description": "Geographic/language locale value, where the format is a combination of an <a href=\"https://en.wikipedia.org/wiki/ISO_639-1\">ISO 639-1</a> code and an optional <a href=\"https://en.wikipedia.org/wiki/ISO_3166-1\">ISO 3166-1</a> code separated by a dash or an underscore.", "required": false, "default": "getLocale()", "type":"string", "values": ["en","de-DE","de-CH","..."]},
+		{"name":"dspLocale", "description": "Locale's display name/output language", "required": false, "default": "getLocaleDisplayName()", "type": "string", "values": ["English","German","French","..."]}
```

Comment:
Why is the "dsp-" prefix better?

Reply:
I used the "dsp" prefix because of http://docs.lucee.org/reference/functions/getlocaleinfo.html#argument-dspLocale (`displayLocale` is not recognized, I've checked).

And the lowercase "output" is because the sentence starts at "Locale's ...", not at "output", and because the previous description also has a lowercase "language" after the slash.
Actually, because the description is not a full sentence, it should be written all in lowercase. Though because the descriptions are often sentences, I still use a capital letter at the beginning.
Having said that, I'm not a native speaker nor grammar specialist. So I might be wrong about the official rules for that. Though this might be another case for #876, so I've added it there.

Reply:
The "output" is quite clear :+1:

Makes all sense. Good idea to add it there.